### DISCUSSION TITLE
Whenever a new version is released, post an announcement in the VRChat group.

### DISCUSSION THREAD

**Discussion by lonelyicer** [vrc-get/vrc-get#1253] [Ideas]
Just like VRChat Open Beta does.
![image](https://github.com/user-attachments/assets/1d07e570-70fb-4309-9327-031af5c23a2f)

**Comment by anatawa12** [comment]
I think it might be good for stable release but for beta releases, I think it's debatable.
In addition, we have to choose tool to create announce from github actions or do manually.

**Reply by lonelyicer** [reply]
It might be possible to create a `github action`-specific VRChat account with group post permissions, and then use the API to create posts

**Reply by lonelyicer** [reply]
[Create a post in a Group](https://vrchatapi.github.io/docs/api/#post-/groups/-groupId-/posts)

**Reply by Spokeek** [reply]
I'm up for the stable releases 👍 

For the github actions, don't you already have a similar repo with https://github.com/anatawa12/sh-actions ?

But we can either make one custom for our use, or if really required, make a seperate repo under the vrc-get group.
